packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## NEXT
+## 0.6.15
 
 * Updates minimum Flutter version to 3.3.
 * Aligns Dart and Flutter SDK constraints.
 * Replace `describeEnum` with the `name` getter.
+* Support custom rendering of those tags without children.
```

Comment:
Please follow the style in [the guide linked from the PR checklist above](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style)